site/locale/de/messages.po

```diff
@@ -1021,7 +1021,7 @@ msgstr ""
 
 #: components/pages/Pledge/InvitationModals/AcceptModal/index.tsx:153
 msgid "pledge.modal.confirm"
-msgstr ""
+msgstr "Bestätigen"
```

Comment:
Is it possible to remove the changes on the locale files?
You did not change any strings in this PR, correct?

Reply:
I think the rebase took care of all the non english files.